frontend/packages/knative-plugin/integration-tests/support/commands/hooks.ts

```diff
@@ -7,6 +7,7 @@ import {
   navigateTo,
   perspective,
 } from '@console/dev-console/integration-tests/support/pages';
+// import { userLoginPage } from '../pages/dev-perspective/common';
```

Comment:
Shouldn't we remove unused code?

Reply:
Just tried something, will remove in the final cut